src/webgl/interaction.js

```diff
@@ -229,7 +229,7 @@ p5.prototype.orbitControl = function(
         deltaPhi = sensitivityY * this.movedY / scaleFactor;
       } else if (this.mouseButton === this.RIGHT) {
         moveDeltaX = this.movedX;
-        moveDeltaY = (this.movedY * -1);
+        moveDeltaY = (this.movedY * (cam.projMatrix.mat4[5] < 0 ? 1 : -1));
```

Comment:
Is there a reason for accessing the projection matrix rather than using `cam.yScale`?

Reply:
Sorry, i just confused with the y-scale. now i have done the changes. it's working fine as you told

Reply:
I guess both have the same sign?

Reply:
Right, I think the yScale value is used in creating the projection matrices. In the end I think they should be equivalent, but it's maybe just more readable to see a value like `yScale` than an index into a matrix.